### PR TITLE
FAI-731 New provider start page 

### PR DESCRIPTION
**Pull request by balaji-j**
Story Link:
https://skillsfundingagency.atlassian.net/browse/FAI-812

Commit Details:
1. Configured UseDfESignIn Switch to render the content in the startup page
2. Unit test included.
3. Startup Page content updated matching Copy Doc

### CODE REVIEW

File: src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+using SFA.DAS.ProviderApprenticeshipsService.Web.Models;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests.Controllers
+{
+    [TestFixture]
+    public class HomeControllerTest
+    {
+        private Web.Controllers.HomeController _sut;
+
+        [Test]
```
Comment (dashton82) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs:14]: the extra `[Test]` attribute can be removed

File: src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            // sut
+            var result = _sut.Index();
+
+            var vr = result as ViewResult;
+
+            // assert
+            vr.Should().NotBeNull();
+
+            var vm = vr.Model as HomeViewModel;
+            vm.Should().NotBeNull();
```
Comment (dashton82) [inline src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs:35]: Should we also assert that the flag is set correctly on the view model?

**Review by dashton82 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs (hunk 2 above)
Comment (balaji-j) [reply src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs:34]: Also, added Assertion to check the UseDfESignIn flag value in the view model.

File: src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs (hunk 1 above)
Comment (balaji-j) [reply src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs]: as suggested, removed the extra [Test] attribute

**Review by dashton82 — APPROVED** [review]

**Pull request closed without merging**